src/test/model-viewer-element-base-spec.js

```diff
@@ -93,7 +97,7 @@ suite('ModelViewerElementBase', () => {
         elements.forEach(e => e.remove());
       });
 
-      test('only models visible in the viewport', async () => {
+      test.skip('only models visible in the viewport', async () => {
```

Comment:
Try as I might, I could not get this test to pass in Edge during CI. If I do a live test on Sauce, it passes fine. I'm going to propose that we skip it until we have a better understanding of why Intersection Observer behaves the way it does on Edge.

Reply:
:+1: